draft-toutain-schc-access-control-00.md

```diff
@@ -69,85 +78,118 @@ Figure {{Fig-archi-overview}} presents the management part of the SCHC architect
-(-----)  read   +====+    +==+==+   +===+===+   +=========+
-( SoR )<------> | RM |<-->|Mgmt.|<==|Access |<==|Other end|<=== 
-(-----) update  +====+    |req. |   |Control|   |auth.    | Managemet.
-        delete            |proc.|   +=======+   +=========+ Request
-                          +=====+                           NETCONF,
-                                                            RESTCONF
-                                                            or CORECONF
+(-----)  read  +====+    +==+==+   +===+===+   +=========+
+( SoR )<------>| RM |<-->|Mgmt.|<==|Access |<==|Other end|<=== 
+(-----) update +====+    |req. |   |Control|   |auth.    | Management
+        delete           |proc.|   +=======+   +=========+ Request
+                         +=====+                           NETCONF,
+                                                           RESTCONF
+                                                        or CORECONF
 ~~~~~~
 {: #Fig-archi-overview title='Overview of management architecture.'}
 
 
-When a management request arrives on a SCHC end-point several processes should be passed before effectively create or update a Rule:
+When a management request arrives on a SCHC endpoint, several processes should be passed before effectively creating or updating a Rule:
 
 1. Other end authentication: the identity of the requester must be verified:
-   * this can be implicit, for example a LPWAN device that receives it from the SCHC core. Hence, authentication is done at Layer 2.
-   * this can be a L2 address. In a LoRaWAN network, for example the DevEUI allows the SCHC core to identify the device.
-   * IP addresses may also be used as well as cryptographic keys.
+   * This can be implicit, for example, an LPWAN device that receives it from the SCHC core. Hence, authentication is done at Layer 2.
+   * This can be an L2 address. In a LoRaWAN network, for example, the DevEUI allows the SCHC core to identify the device.
+   * IP addresses may also be used, as well as cryptographic keys.
 2. Access control: Once authenticated, the associated Set of Rules of the instance is retrieved. 
-   * these rules are enriched with access control information that will be defined in this document. 
-   * if the Set of Rules does not contain any access control information, the end-point is not allowed to modify the Rules content.
+   * These rules are enriched with access control information that will be defined in this document. 
+   * If the Set of Rules does not contain any access control information, the end-point is not allowed to modify the Rules' content.
 3. Management request processing: The NETCONF, RESTCONF or CORECONF request is processed and passed to the end-point Rule Manager.
-5. The Rule Manager (RM) applies the changes (create, read, update or delete) to the Set of Rules (SoR) data base. 
+5. The Rule Manager (RM) applies the changes (create, read, update, or delete) to the Rule database. 
```

Comment:
Will we keep data base or just set of rules?, just to change or keep the image as SoR, I'd rather use SoR